MIGRATION_GUIDE_V2.md

```diff
@@ -0,0 +1,105 @@
+# Ember-sortable
+
+## Migration Guide (v1 -> v2)
+
+### Higher Order Components
+`Ember-sortable` can now be built using `higher order components`
```

Comment:
is it possible to add link to some article about high order components in ember?

Reply:
I can do that, no problem.

Reply:
@chriskrycho 👋 
Just letting you know since emberjs.com doesn't have an up-to-date doc on this, I'm linking your beautiful article on HOC since yours is the most recent one I can find heh.

Reply:
https://v4.chriskrycho.com/2018/higher-order-components-in-emberjs.html